spec/system/index_spec.rb

```diff
@@ -4,10 +4,16 @@
 
 RSpec.describe 'Index', type: :system do
   describe 'index page' do
-    it 'contains smartflix' do
+    it 'contains Smartflix' do
       get '/'
 
       expect(response.body).to include('Smartflix')
     end
+
+    it 'contains movie titles' do
```

Comment:
you can add another spec if you provide `limit` parameter that its showing only limit amount of tites in the body. Also you can add another spec where you test the default behaviour of the limit variable. Where its only only 10 titles